admin_guide/allocating_node_resources.adoc

```diff
@@ -0,0 +1,115 @@
+= Allocating Node Resources
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+To provide more reliable scheduling and minimize node resource overcommitment,
+each node can reserve a portion of its resources for use by all underlying
+link:../architecture/infrastructure_components/kubernetes_infrastructure.html#node[node
+components] (e.g., kubelet, kube-proxy, Docker) and the remaining system
+components (e.g., *sshd*, *NetworkManager*) on a node host. Once specified, the
+scheduler has more information about the resources (e.g., memory, CPU) a node
+has allocated for pods.
+
+[[allocating-node-settings]]
+== Configuring Nodes for Allocated Resources
+
+Resources reserved for node components are based on two node settings:
+
+[options="header",cols="1,2"]
+|===
+
+|Setting |Description
+
+|`*kube-reserved*`
+| A set of `<resource_type>=<resource_quantity>` pairs (e.g.,
+`cpu=200m,memory=30G`) that describe resources reserved for node components.
+Default is none.
+
+|`*system-reserved*`
+| A set of `<resource_type>=<resource_quantity>` pairs (e.g.,
+`cpu=200m,memory=30G`) that describe resources reserved for the remaining system
+components. Default is none.
+|===
+
+You can specify values for these settings in the `*kubeletArguments*` section of
+the
+link:../install_config/master_node_configuration.html#node-configuration-files[node
+configuration file] (the *_/etc/origin/node/node-config.yaml_* file by default).
+Add the section if it does not already exist:
+
+.Node Allocatable Resources Settings
+====
+[source,yaml]
+----
+kubeletArguments:
+  kube-reserved:
+    - "cpu=200m,memory=30G"
+  system-reserved:
+    - "cpu=200m,memory=30G"
+----
+====
+
+Currently, the `*cpu*` and `*memory*` resource types are supported. For `*cpu*`,
+the resource quantity is specified in units of cores (e.g., 200m, 100Ki, 50M).
+For `*memory*`, it is specified in units of bytes (e.g., 200Ki, 100M, 50Gi).
+
+See link:../dev_guide/compute_resources.html[Compute Resources] for more
+details.
+
+If a flag is not set, it defaults to *0*. If none of the flags are set, the
+allocated resource is set to node's capacity as it was before the introduction
+of allocatable resources.
+
+[[computing-allocated-resources]]
+== Computing Allocated Resources
+
+An allocated amount of a resource is computed based on the following formula:
+
+----
+[Allocatable] = [Node Capacity] - [kube-reserved] - [system-reserved]
+----
+
+If `[Allocatable]` is negative, it is set to *0*.
+
+[[viewing-node-allocatable-resources-and-capacity]]
+== Viewing Node Allocatable Resources and Capacity
+
+To see node's current capacity and allocatable resources, you can run:
+
+====
+----
+$ oc get node/<node_name> -o yaml
```

Comment:
Switched this back to YAML. Trying to stick with YAML across the docs unless it really needs to be JSON for some reason.